blocks/social-share/social-share.js

```diff
@@ -0,0 +1,36 @@
+import { loadScript } from '../../scripts/aem.js';
+import { getLocaleInfo } from '../../scripts/scripts.js';
+
+export default async function decorate(block) {
+
+async function loadSocialShare() {
+  loadScript('https://platform-api.sharethis.com/js/sharethis.js#property=5cae6f10a8698b001266ffd9&product=inline-share-buttons');
+
+    const localeInfo = getLocaleInfo();
+
+    const socialSharePromise = fetch(`${localeInfo.urlPrefix}/blogs/fragments/social-share.plain.html`);
```

Comment:
why do we need to load a fragment?
if I understand correctly all that we need is a `div` with the class `sharethis-inline-share-buttons`. correct? do we need something else?

which means, you don't need to load a fragment, you can simply do:
```javascript
block.classList.add('sharethis-inline-share-buttons');
```
correct?